cmd/minikube/cmd/start.go

```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```

Comment:
LGTM :+1: , I am not sure we can add any test case for this change because that's a print statement.